xtreemfs-common/Dockerfile

```diff
@@ -12,9 +12,11 @@ RUN apt-get -qy install \
     maven \
     openjdk-8-jdk
 
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
-ENV JAVA_INCLUDE_PATH /usr/lib/jvm/java-8-openjdk-amd64/include
-ENV JAVA_INCLUDE_PATH2 /usr/lib/jvm/java-8-openjdk-amd64/include/linux
+RUN JAVA_HOME=$(java -XshowSettings:properties -version 2>&1 > /dev/null | grep 'java.home' | grep -oP "/.*")
+
+ENV JAVA_HOME ${JAVA_HOME}
```

Comment:
This line is unnecessary.

Reply:
> This line is unnecessary.

I am not able to remove the local variable stored by running the RUN command. I'm not a bash expert, I tried to remove the line by doing this:
![2019-07-04 01_03_24-dockerfile_removed_line](https://user-images.githubusercontent.com/3049704/60630056-b378cf00-9df8-11e9-8526-f16c9519d12b.png)

And got this result:


I tried replacing the brackets "(" ")" by braces "{" "}" but didn't work too. If I'm not wrong you was suggesting removing the first "RUN ..." line, can you give me more clues about the way to proceed plz ? Thanks :)